sw/device/lib/crypto/drivers/kmac.c

```diff
@@ -545,6 +548,14 @@ static status_t kmac_write_key_block(kmac_blinded_key_t *key) {
       KMAC_KEY_LEN_REG_RESVAL, KMAC_KEY_LEN_LEN_FIELD, key_len_enum);
   abs_mmio_write32(kKmacBaseAddr + KMAC_KEY_LEN_REG_OFFSET, key_len_reg);
 
+  // Write random words to the key registers.
+  for (size_t i = 0; i * sizeof(uint32_t) < key->len; i++) {
+    abs_mmio_write32(kKmacKeyShare0Addr * i * sizeof(uint32_t),
```

Comment:
(small typo, this and next line)

```suggestion
    abs_mmio_write32(kKmacKeyShare0Addr + i * sizeof(uint32_t),
    
```